mappings/net/minecraft/client/gl/GlFramebuffer.mapping

```diff
@@ -0,0 +1,43 @@
+CLASS none/bpf net/minecraft/client/gl/GlFramebuffer
+	FIELD a texWidth I
+	FIELD b texHeight I
+	FIELD c viewWidth I
+	FIELD d viewHeight I
+	FIELD e useDepthAttachment Z
+	FIELD f fbo I
+	FIELD g colorAttachment0 I
```

Comment:
Maps more directly to OpenGL, and in the event that Mojang ever decides to use more than one colour attachment - which is very much plausible as they may need to output colour and normal information in a single pass - we are prepared.